### PR TITLE
fix component enable error and upgrade image version

### PR DESCRIPTION
**Pull request by cywang1905**

### CODE REVIEW

File: doc/en/lazy_load.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
   image:
     pullPolicy: Always
     repository: docker.io/slimeio/slime-lazyload
-    tag: v0.2.0-alpha
+    tag: v0.2.1
```
Comment (YonkaFang) [inline doc/en/lazy_load.md:28]: REPLACE_WITH_TAG_OR_COMMIT

File: install/samples/lazyload/easy_install_lazyload.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```
Comment (YonkaFang) [inline install/samples/lazyload/easy_install_lazyload.sh:1]: tag_or_com=${TAG_OR_COMMIT}
if [[ -z "${tag_or_com}" ]]; then
  echo "should pass tag or commit via env TAG_OR_COMMIT" >&2
  exit 1
fi

Comment (cywang1905) [reply install/samples/lazyload/easy_install_lazyload.sh:1]: Done. Use latest_tag as default, user can add paramater to override, and use customize tag or commit version.

File: doc/en/lazy_load.md (hunk 1 above)
Comment (cywang1905) [reply doc/en/lazy_load.md]: Done

**Comment by cywang1905** [comment]
Now the latest_tag info is only contained in yaml files, in the directory of ./install. All markdown files are no need to change due to new latest_tag.

File: doc/en/lazy_load.md [hunk 3]
```diff
@@ -25,17 +30,17 @@ spec:
...
     pullPolicy: Always
     repository: docker.io/slimeio/slime-lazyload
-    tag: v0.2.0-alpha
+    tag: {{your_lazyload_tag}}
   module:
     - name: lazyload
       fence:
         enable: true
         wormholePort: 
           - "9080" # replace to your application service ports
```
Comment (YonkaFang) [inline doc/en/lazy_load.md:39]: "and extend the list in case of multi ports"

File: doc/en/lazy_load.md [hunk 4]
```diff
@@ -25,17 +30,17 @@ spec:
...
-          - {{your port}}
+          - {{your_port}}
```
Comment (YonkaFang) [inline doc/en/lazy_load.md:40]: remove this line

File: doc/en/lazy_load.md [hunk 5]
```diff
@@ -47,7 +52,7 @@ spec:
       type: namespaced
       namespace:
         - default # replace to or add your deployment's namespace
```
Comment (YonkaFang) [inline doc/en/lazy_load.md:54]: refer to "wormholePort"

File: install/samples/lazyload/easy_install_lazyload.sh [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
+if [[ "$#" -eq 0 ]]; then
+  tag_or_commit=$(curl -s https://api.github.com/repos/slime-io/slime/tags | grep 'name' | cut -d\" -f4 | head -1)
+  echo "No specified tag or commit. Use latest tag: $tag_or_commit."
+else
+  tag_or_commit=$1
+  echo "Use specified tag or commit: $tag_or_commit"
+fi
+
```
Comment (YonkaFang) [inline install/samples/lazyload/easy_install_lazyload.sh:9]: add check for empty tag_or_commit

File: install/samples/lazyload/easy_uninstall_lazyload.sh [hunk 7]
```diff
@@ -1,6 +1,18 @@
...
+else
+  tag_or_commit=$1
+  echo "Use specified tag or commit: $tag_or_commit"
+fi
+
+crds_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/crds.yaml"
+deployment_slimeboot_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/deployment_slime-boot.yaml"
+slimeboot_lazyload_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/samples/lazyload/slimeboot_lazyload.yaml"
+
 for i in $(kubectl get ns);do kubectl delete servicefence -n $i --all;done
```
Comment (YonkaFang) [inline install/samples/lazyload/easy_uninstall_lazyload.sh:14]: --all-namespaces ?

File: install/samples/smartlimiter/easy_uninstall_limiter.sh [hunk 8]
```diff
@@ -1,6 +1,18 @@
...
+else
+  tag_or_commit=$1
+  echo "Use specified tag or commit: $tag_or_commit"
+fi
+
+crds_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/crds.yaml"
+deployment_slimeboot_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/init/deployment_slime-boot.yaml"
+slimeboot_smartlimiter_url="https://raw.githubusercontent.com/slime-io/slime/$tag_or_commit/install/samples/smartlimiter/slimeboot_smartlimiter.yaml"
+
 for i in $(kubectl get ns);do kubectl delete smartlimiter -n $i --all;done
```
Comment (YonkaFang) [inline install/samples/smartlimiter/easy_uninstall_limiter.sh:14]: --all-namespaces ?

File: install/samples/lazyload/easy_uninstall_lazyload.sh (hunk 7 above)
Comment (cywang1905) [reply install/samples/lazyload/easy_uninstall_lazyload.sh:14]: --all means delete all svf in single namespace

File: install/samples/smartlimiter/easy_uninstall_limiter.sh (hunk 8 above)
Comment (cywang1905) [reply install/samples/smartlimiter/easy_uninstall_limiter.sh:14]: same with svf

File: doc/en/lazy_load.md (hunk 3 above)
Comment (cywang1905) [reply doc/en/lazy_load.md]: Done.

File: doc/en/lazy_load.md (hunk 4 above)
Comment (cywang1905) [reply doc/en/lazy_load.md]: Done.

File: doc/en/lazy_load.md (hunk 5 above)
Comment (cywang1905) [reply doc/en/lazy_load.md]: Done.

File: install/samples/lazyload/easy_install_lazyload.sh (hunk 6 above)
Comment (cywang1905) [reply install/samples/lazyload/easy_install_lazyload.sh:14]: Done.

File: install/samples/lazyload/easy_uninstall_lazyload.sh (hunk 7 above)
Comment (cywang1905) [reply install/samples/lazyload/easy_uninstall_lazyload.sh:19]: https://stackoverflow.com/questions/33509194/command-to-delete-all-pods-in-all-kubernetes-namespaces

**Review by YonkaFang — APPROVED** [review]

**Pull request merged**